app/src/main/java/com/example/gravit/splash/SplashActivity.kt

```
@@ -20,69 +32,256 @@ import androidx.compose.ui.platform.LocalContext
+    }
+
+    val webIntent = Intent(
+        Intent.ACTION_VIEW,
+        "https://play.google.com/store/apps/details?id=$packageName".toUri()
+    ).apply {
+        addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+    }
+
+    try {
+        context.startActivity(playStoreIntent)
+    } catch (e: ActivityNotFoundException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)